src/main/resources/default/assets/common/scripts/overlay.js.pasta

```diff
@@ -73,7 +73,8 @@
         _overlay = _overlayContainer.firstChild;
 
         if (closable) {
-            _overlay.querySelector('.sci-overlay-button-close-js').addEventListener('click', function () {
+            const _closeButton = _overlay.querySelector('.sci-overlay-button-close-js');
+            sirius.addClickOrEnterListener(_closeButton, function () {
```

Comment:
Haben wir eine Policy bzgl. der Syntax `function () { … }` vs `() => { … }`? Ist ja wegen des `this` Pointers nicht das gleiche, und in den meisten Fällen erscheint mir die Pfeilsyntax eigentlich als die "richtigere", da eben keine komischen Seiteneffekte. Oder muss das alles hier noch IE11-kompatibel sein?